.github/ISSUE_TEMPLATE/--questions.md

```diff
@@ -7,7 +7,7 @@ labels: question
 ---
 
 <!-- ⚠️⚠️ Do Not Delete These Comments. ⚠️⚠️ -->
-<!-- Read our Code of Conduct: https://github.com/Open-Source-Community-VIT-AP/Hacktoberfest-2020/blob/master/.github/CODE_OF_CONDUCT.md -->
+<!-- Read our Code of Conduct: https://github.com/Open-Source-Community-VIT-AP/Hacktoberfest-2020/blob/Updates/.github/CODE_OF_CONDUCT.md -->
```

Comment:
This should be master, if I'm not wrong.

Reply:
Yeah, made a mistake. Thanks for the review.